src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/RequestHandler.java

```diff
@@ -469,7 +469,7 @@ public static void cleanRequestData(final IStandardRequestManager manager, final
      * @param token The token to query
      * @throws IllegalArgumentException when the token is unknown to the given manager.
      */
-    public static IRequest getRequest(final IStandardRequestManager manager, final IToken token) throws IllegalArgumentException
+    public static IRequest getRequest(final IStandardRequestManager manager, final IToken<?> token) throws IllegalArgumentException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)